pkgs/by-name/lu/lubelog/package.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, buildDotnetModule
+, dotnetCorePackages
+, fetchFromGitHub
+}:
+
+buildDotnetModule {
+  pname = "lubelog";
+  version = "1.1.5";
+
+  src = fetchFromGitHub {
+    owner = "hargata";
+    repo = "lubelog";
+    rev = "f0869c741f1bd5e45fdeda8b518d625ae0b8d2f5";
+    hash = "sha256-d71bVsXB4VvNWADzTTqPqsH6/6W9wo1ynFR069eF9a8=";
+  };
+
+  projectFile = "CarCareTracker.sln";
+  nugetDeps = ./deps.nix; # File generated with `nix-build -A package.passthru.fetch-deps`.
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.aspnetcore_8_0;
+
+  makeWrapperArgs = [
+    "--set DOTNET_CONTENTROOT ${placeholder "out"}/lib/lubelog"
+  ];
+
+  executables = [ "CarCareTracker" ]; # This wraps "$out/lib/$pname/foo" to `$out/bin/foo`.
+
+  packNupkg = true; # This packs the project as "foo-0.1.nupkg" at `$out/share`.
+
+  meta = with lib; {
+    description = "Vehicle Service Records and Maintainence Tracker";
+    longDescription = ''
+      Self-Hosted, Open-Source, Unconventionally-Named Vehicle Maintenance Records and Fuel Mileage Tracker
```

```suggestion
    description = "Vehicle service records and maintainence tracker";
    longDescription = ''
      Self-hosted, open-source, unconventionally-named vehicle maintenance records and fuel mileage tracker
```

It's not a cryptonym or a codename, it's just description and should be written as description - first letter uppercase, without trailing dot. https://nixos.org/manual/nixpkgs/stable/#var-meta-description